### PR TITLE
UINV-468: Open version history pane on click change log icon

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[UINV-468](https://folio-org.atlassian.net/browse/UINV-468): Open version history pane on click change log icon
## Approach

https://github.com/user-attachments/assets/c4570ea7-b37c-4b25-966a-10feb5cb672b

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/invoices/InvoiceDetails/InvoiceDetails.js [hunk 1]
```diff
@@ -89,6 +100,17 @@ function InvoiceDetails({
   exportFormat,
   refreshData,
 }) {
+  const history = useHistory();
+  const stripes = useStripes();
+  const { search } = useLocation();
+  const accordionStatusRef = useRef();
+  const { restrictions, isLoading: isRestrictionsLoading } = useAcqRestrictions(
+    invoice.id, invoice.acqUnitIds,
```
Comment (Terala-Priyanka) [inline src/invoices/InvoiceDetails/InvoiceDetails.js:108]: Could you destructor invoice to obtain id, acqUnitIds, status as it is referred several times in this module

Comment (alisher-epam) [reply src/invoices/InvoiceDetails/InvoiceDetails.js]: sure

**Review by Terala-Priyanka — APPROVED** [review]
There are more to 'invoice.' references. Although they are not a result of your changes, could you refactor them on the same lines?

File: src/common/constants/constants.js
Comment (usavkov-epam) [inline src/common/constants/constants.js:1]: Could you please create a separate file for routes? e.g. `routes.js`

File: src/invoices/InvoicesList/InvoicesList.js [hunk 2]
```diff
@@ -57,6 +57,8 @@ import {
   searchableIndexes,
 } from './InvoicesListSearchConfig';
 import { ExportSettingsModal } from './ExportSettingsModal';
+import { INVOICE_LINE_VIEW_ROUTE, INVOICE_VERSION_HISTORY_ROUTE, INVOICE_VIEW_ROUTE } from '../../common/constants';
```
Comment (usavkov-epam) [inline src/invoices/InvoicesList/InvoicesList.js:60]: plz break into separate lines and align the order of imports

**Review by usavkov-epam — COMMENTED** [review]
Could you please enable a list of snapshot tests in the scope of your PR?
- https://github.com/folio-org/ui-invoice/blob/69e750c7d9e9ad46efea727fc7d65ada16c1a76e/src/invoices/AdjustmentsForm/AdjustmentsForm.test.js#L47-L48
- https://github.com/folio-org/ui-invoice/blob/69e750c7d9e9ad46efea727fc7d65ada16c1a76e/src/invoices/AdjustmentsForm/AdjustmentsForm.test.js#L56-L57
- https://github.com/folio-org/ui-invoice/blob/69e750c7d9e9ad46efea727fc7d65ada16c1a76e/src/invoices/InvoicesList/InvoicesListFilters/InvoicesListFilters.test.js#L38-L39
- https://github.com/folio-org/ui-invoice/blob/69e750c7d9e9ad46efea727fc7d65ada16c1a76e/src/invoices/InvoiceLineForm/InvoiceLineForm.test.js#L69-L70
- https://github.com/folio-org/ui-invoice/blob/69e750c7d9e9ad46efea727fc7d65ada16c1a76e/src/invoices/InvoiceForm/InvoiceForm.test.js#L87-L88

Those were release blockers

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**